auth/openid_connect.adoc

```diff
@@ -0,0 +1,219 @@
+
+[[openid-connect]]
+== OpenID-Connect
+
+External Authentication on the Appliance has been enhanced to support OpenID-Connect,
+which will also be referred to as _OIDC_.
+
+The OIDC implementation has been tested with KeyCloak 3.2.1.
+
+In this guide we will cover how to manually configure an Appliance's
+external authentication to work with OIDC. The OIDC implementation has been
+tested with KeyCloak but is implemented generically using Apache's mod_auth_openidc
+module and should work with other OIDC Identity Providers.
+
+The current implementation only secures the Appliance's Web administrative UI with OIDC.
+
+*Note:* The REST API and Self-Service UI do not currently support OIDC.
+
+1.  <<requirements, Requirements>>
+2.  <<configuring-oidc, Configuring OIDC>>
+3.  <<configuring-admin-ui, Configuring the Administrative UI>>
+4.  <<re-enabling-local-login, Re-Enabling Local Login>>
+
+[[requirements]]
+== Requirements
+
+The following is needed in order to enable OIDC Authentication to the Appliance:
+
+* A CentOS/RHEL 7.5 based Appliance
+* A OIDC Identity Provider, (e.g. KeyCloak 3.2.1 or Later)
+
+[[configure-oidc]]
+== Configure OIDC 
+
+First ssh to the appliance as root, then install the mod_auth_openidc package:
+
+```
+# yum install -y mod_auth_openidc
```

Comment:
why ?  we should include this as we do with other apache modules.

Reply:
Good point.
I've added mod_auth_openidc to the list of packages to be built in.

See: https://github.com/ManageIQ/manageiq-appliance-build/pull/272